3scale-image-streams.yml

```diff
@@ -24,8 +24,6 @@ items:
           will automatically update to use the latest version of 3Scale APIcast API Gateway
           available on OpenShift, including major versions updates.
         tags: api,gateway,3scale
-      referencePolicy:
```

Comment:
Well, it was not just the name of the registry. IIRC this `referencePolicy` was needed by the library. Anyway, I would have waited from @bparees confirmation before merging that as this constantly reloaded inside their cluster.

Reply:
What the referencePolicy does is that it allows you to configure from where the external registry images are pulled. It will not prevent OpenShift from working correctly. You have more information in:
https://docs.openshift.com/online/dev_guide/managing_images.html#reference-policy